logbook-spring-boot-autoconfigure/pom.xml

```
@@ -64,6 +64,17 @@
             <artifactId>logbook-httpclient5</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.zalando</groupId>
+            <artifactId>logbook-openfeign</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>io.github.openfeign</groupId>
+            <artifactId>feign-core</artifactId>
+            <version>${feign.version}</version>
+            <scope>optional</scope>
```

Thanks for updating this. Maven has 6 scopes, but `optional` is [not one of them](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#dependency-scope). Optional is actually a flag, which can be true or false.
```suggestion
            <optional>true</optional>
```